app/assets/stylesheets/modules/_global-nav.scss

```diff
@@ -0,0 +1,21 @@
+.global-nav {
+  background-color: $base-dark-accent-color;
+  height: em(70);
+  padding: 1em 0;
+  position: relative;
+}
+
+.logo a {
+  color: $white;
+  float: left;
```

Comment:
Properties should be ordered color, display, float, font-weight, letter-spacing, padding-top